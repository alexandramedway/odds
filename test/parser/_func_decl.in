do sum = (x, y) ->
    do print(x)
    do print(y)
    return x + y
