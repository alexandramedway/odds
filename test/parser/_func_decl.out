[
    Do(
        Assign(
            Id(sum),
            Fdecl({
                params=[
                    Id(x) ;
                    Id(y)
                ] ;
                body=[
                    Do(Call(print, [Id(x)])) ;
                    Do(Call(print, [Id(y)]))
                ] ;
                return=Binop(Id(x), Add, Id(y))
            })
        )
    )
]
